src/components/Scheme/SchemeProposalsPage.tsx

```diff
@@ -134,20 +135,36 @@ class SchemeProposalsPage extends React.Component<IProps, null> {
             <div className={css.proposalsHeader}>
               No upcoming proposals
             </div>
-            <p>You can be the first one to create a {schemeFriendlyName} proposal today! :)</p>
-            <div className={css.cta}>
-              <Link to={"/dao/" + daoState.address}>
-                <img className={css.relax} src="/assets/images/lt.svg"/> Back to schemes
-              </Link>
-              <a className={classNames({
-                [css.blueButton]: true,
-                [css.disabled]: !isActive,
-              })}
-              href="#!"
-              onClick={isActive ? this._handleNewProposal : null}
-              data-test-id="createProposal"
-              >+ New Proposal</a>
-            </div>
+            { isWikiScheme ? 
+              <>
```

Comment:
what does `<>` do here?

Reply:
It's just an empty html element, to wrap the entire section. It literally does nothing but make react to not compile, this way we can have the `<p>` and `<div>` wrapped in an element